frontend/context/BalanceProvider.tsx

```diff
@@ -118,95 +108,13 @@ export const BalanceProvider = ({ children }: PropsWithChildren) => {
     }, 0);
   }, [stakedBalances]);
 
-  /**
-   * An array of wallet addresses that have low native token balances
-   */
-  const lowBalances = useMemo(() => {
-    const result: {
-      serviceConfigId: string;
-      chainId: ChainId;
-      walletAddress: Address;
-      balance: number;
-      expectedBalance: number;
-    }[] = [];
-
-    if (!services || !wallets || !walletBalances) return result;
-
-    for (const service of services) {
-      const serviceId = service.service_config_id;
-      const serviceHomeChainId = service.home_chain_id;
-
-      const stakedBalancesForService = stakedBalances.filter(
-        (balance) =>
-          balance.serviceId === serviceId &&
-          balance.chainId === serviceHomeChainId,
-      );
-
-      if (stakedBalancesForService.length === 0) continue;
-
-      // Get addresses for master safe and agent home chain safe
-      const masterSafeWallet = wallets.find(
-        (wallet) =>
-          wallet.owner === WalletOwnerType.Master &&
-          wallet.type === WalletType.Safe &&
-          wallet.chainId === serviceHomeChainId,
-      );
-
-      const masterSafeAddress = masterSafeWallet?.address;
-      const stakedAgentSafeAddress =
-        service.chain_configs[serviceHomeChainId]?.chain_data.multisig;
-
-      if (!masterSafeAddress && !stakedAgentSafeAddress) continue;
-
-      const masterSafeBalanceResult = walletBalances.find(
-        (balance) =>
-          balance.walletAddress === masterSafeAddress && balance.isNative,
-      );
-
-      const stakedAgentSafeBalanceResult = walletBalances.find(
-        (balance) =>
-          balance.walletAddress === stakedAgentSafeAddress && balance.isNative,
-      );
-
-      if (
-        masterSafeBalanceResult &&
-        masterSafeBalanceResult.balance < LOW_MASTER_SAFE_BALANCE
-      ) {
-        result.push({
-          serviceConfigId: service.service_config_id,
-          chainId: serviceHomeChainId,
-          walletAddress: masterSafeBalanceResult.walletAddress,
-          balance: masterSafeBalanceResult.balance,
-          expectedBalance: LOW_MASTER_SAFE_BALANCE,
-        });
-      }
-
-      if (
-        stakedAgentSafeBalanceResult &&
-        stakedAgentSafeBalanceResult.balance < LOW_AGENT_SAFE_BALANCE
-      ) {
-        result.push({
-          serviceConfigId: service.service_config_id,
-          chainId: serviceHomeChainId,
-          walletAddress: stakedAgentSafeBalanceResult.walletAddress,
-          balance: stakedAgentSafeBalanceResult.balance,
-          expectedBalance: LOW_AGENT_SAFE_BALANCE,
-        });
-      }
-    }
-
-    return result;
-  }, [services, stakedBalances, wallets, walletBalances]);
-
-  const isLowBalance = useMemo(() => lowBalances?.length > 0, [lowBalances]);
```

Comment:
is this temporarily removed? there are a lot of dependencies on this in the code

Reply:
Yeah, will be added back for sure